app/views/moderations/_mod_sidebar_left.html.erb

```diff
@@ -40,4 +34,17 @@
       </span>
     </a>
   </div>
+  <% if @community_mod_channel %>
+    <div class="fs-s mod-feedback">
+      <span class="fw-medium">Have feedback to improve your Mod experience?</span>
+      <span>Please share in
+        <a
+          class="mod-feedback-link"
+          href="/connect/<%= @community_mod_channel.slug %>"
+          target="_blank"
+          rel="noopener noreferrer nofollow">
+          <%= @community_mod_channel.channel_name %></a>.
```

Comment:
I think we can do "your Connect Channel!" in plain text here with no link, and then in the "Resources" section we have a link to "Connect Channel" which would link to this specific channel.

Reply:
ok!